autoscale/src/main/java/com/sequenceiq/periscope/controller/DistroXAutoScaleClusterV1Controller.java

```diff
@@ -138,6 +146,12 @@ private DistroXAutoscaleClusterResponse createClusterJsonResponse(Cluster cluste
         return distroXAutoscaleClusterResponseConverter.convert(cluster);
     }
 
+    private Cluster updateAutoscaleState(Long clusterId, AutoscaleClusterState autoscaleState) {
+        Cluster updatedCluster = asClusterCommonService.setAutoscaleState(clusterId, autoscaleState);
```

Comment:
Can we possibly capture the user, etc details - as to who made this change?
Independent of this CR: Does that need to be captured into AuditEvents?

Reply:
user details belongs to audit events and these are already captured in auditevents.